### DISCUSSION TITLE
Image zoom level too low

### DISCUSSION THREAD

**Discussion by mztiq** [nunocoracao/blowfish#335] [Q&A] [answered]
Hi,

when I add an image with the [Figure Shortcode](https://nunocoracao.github.io/blowfish/docs/shortcodes/#figure) via `![Alt text](image.jpg "Image caption")` the image zoom level when clicking on it most of the time is very low to the point it barely increases the size of the linked image.  
When I add an image with the size 2536 x 1215 for exampl it barely increases its size when I click on it (see the first picture in [this post](https://dizzytech.de/posts/bookstack/) as an example).  
Am I doing something wrong or is there a way around this problem?  

Any hints are appreciated, thanks in advance.

**Comment by nunocoracao** [comment, marked as answer]
@mztiq approved. can you open up the ticket for it? (as a bug)

**Reply by mztiq** [reply]
Done.  
Thanks for taking the time.
